### DISCUSSION TITLE
How to get rollout history by k8s restfulapi

### DISCUSSION THREAD

**Discussion by SpringHgui** [kubernetes-client/csharp#1095] [Q&A]
How to get rollout history by k8s restfulapi
```
kubectl rollout xxxxxx
```

**Comment by SpringHgui** [comment]
how to do [rollback-daemon-set](https://kubernetes.io/docs/tasks/manage-daemon/rollback-daemon-set/) by sdk

**Reply by tg123** [reply]
those will be supported in https://github.com/orgs/kubernetes-client/projects/1
